concepts/submission_date.md

```diff
@@ -0,0 +1,77 @@
+# Submission Date
+
+This document will help you understand `submission_date`, why are standardizing on it, and how it
+differs from `activity_date`.
+
+## What are Submission Date and Activity Date
+
+`submission_date` is the server time at which a ping is received from the client. We use it to
+partition many of our data sets.
+
+`activity_date` is the client time at which a subsession started. It is acquired by removing time
+of day from `subsession_start_date`. It is sometimes normalized by throwing away values too far
+from `submission_date`, values not in the years 2000-2099, and values that are not dates.
+
+## Submission Delay
+
+[Chutten posted in his blog](https://chuttenblog.wordpress.com/2017/09/12/two-days-or-how-long-until-the-data-is-in/)
+about how 95% of Firefox data is submitted within 2 days of `activity_date`, since pingsender was
+released. This means that using `submission_date` smears data between today and yesterday
+(mostly). In another
+[chutten blog post](https://chuttenblog.wordpress.com/2017/07/12/latency-improvements-or-yet-another-satisfying-graph/)
+about submission delay a graph shows 95% submission delay to generally sit between 100 and 150
+hours without pingsender.
+
+Only some types of pings are sent by pingsender.
+[Bug 1310703](https://bugzilla.mozilla.org/show_bug.cgi?id=1310703) introduced pingsender for crash
+pings and was merged in Firefox 54, which hit release on June 13, 2017.
+[Bug 1336360](https://bugzilla.mozilla.org/show_bug.cgi?id=1336360) moved shutdown pings to
+pingsender and was merged in Firefox 55, which hit release on August 8, 2017.
+[Bug 1374270](https://bugzilla.mozilla.org/show_bug.cgi?id=1374270) added sending health pings on
+shutdown via pingsender and was merged in Firefox 56, which hit release on Sept 28, 2017.
+Other types of pings are not sent with pingsender. This is usually okay because Firefox is expected
+to continue running long enough to send those pings.
+
+Mobile clients do not have pingsender, so they suffer delay as given in
+[this query](https://sql.telemetry.mozilla.org/queries/49867#134105).
+
+## Why Submission Date
+
+TL;DR
+
+* not subject to client clock skew
+* doesn't require normalization
+* good for backfill
+* good for daily processing
+* and usually good enough
+
+In [bug 1422892](https://bugzilla.mozilla.org/show_bug.cgi?id=1422892) we decided to standardize
+on `submission_date`. From [Comment 1](https://bugzilla.mozilla.org/show_bug.cgi?id=1422892#c1):
+
+> `activity_date` is generally preferable to `submission_date` because it's closer to what we
```

Comment:
Not a strong opinion, but I'd suggest limiting this section to a recommendation and the TLDR, leaving the quote and reasoning in the bug.

Reply:
dropped the bug quotes